gradlew

```diff
@@ -15,6 +15,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we need this? I agreed with Johannes to not demand this for all Gradle-related files (within `LDM`).

Reply:
I didn't touch these files manually, they are the result of `gradle wrapper --gradle-version 8.13`

Reply:
Ah, yes: it's not the Trumpf-ID :) So, no worries